cg/store/api/add.py

```diff
@@ -63,12 +64,12 @@ def add_customer(
     def add_collaboration(self, internal_id: str, name: str, **kwargs) -> Collaboration:
         """Build a new customer group record."""
 
-        return self.Collaboration(internal_id=internal_id, name=name, **kwargs)
+        return Collaboration(internal_id=internal_id, name=name, **kwargs)
```

Comment:
for these functions we can then remove the self parameter in the definition, is this correct?

Reply:
If we want to make them static, yes